snap/info.go

```diff
@@ -415,6 +422,13 @@ func (plug *PlugInfo) String() string {
 	return fmt.Sprintf("%s:%s", plug.Snap.Name(), plug.Name)
 }
 
+func (slot *SlotInfo) Attr(key string) (interface{}, error) {
+	if val, ok := slot.Attrs[key]; ok {
+		return val, nil
+	}
+	return nil, fmt.Errorf("attribute %q not found", key)
+}
+
```

Comment:
It would be great if you could just test this explicitly in a separate test method.